source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+
+You can reference multiple entities at one time by putting them into a **[group](https://www.home-assistant.io/integrations/group/)**
+
+## Automation
+
+**[Automations](https://www.home-assistant.io/docs/automation/)** let you go father by telling Home Assistant to automatically do things without you having to hit a button yourself. For example, you could write an automation that says “turn on the lights when it gets dark if I’m home”.
+
+An automation has three main parts:
+
+- **[Triggers](https://www.home-assistant.io/docs/automation/trigger/)** - something that starts the automation, like the sun going down or a motion sensor noticing movement
+- **[Conditions](https://www.home-assistant.io/docs/automation/condition/)** - let you decide whether to complete the automation. For example: "only if someone is home".
+- **[Action](https://www.home-assistant.io/docs/automation/action/)** - what the automation does, like "turn on lights"
+
+**[Templates](https://www.home-assistant.io/docs/automation/templating/)** let advanced users use variables and logic in their automations
+
+## Events and state
+
+Entities in Home Assistant are controlled by sending **[events](https://www.home-assistant.io/docs/configuration/events/)**.
+
+Home Assistant remembers the **[state](https://www.home-assistant.io/docs/configuration/state_object/)** of each entity. The state means what *mode* or *condition* the entity is in this moment. For example, a light could have two states: ‘on’ and ‘off’, and a garage door could be in one of two states: ‘open’ or ‘closed’.
+
+A **[scene](https://www.home-assistant.io/docs/scene/)** is a list of entities along with the state you want them each to be in. When you trigger that scene, Home Assistant will figure out the right events to get each of those entities into that state.
+
+For example, let’s imagine you have a scene called “welcome home” that says “lights on, door unlocked”. If you turn on that scene, Home Assistant will check if your lights are on and turn them on if they’re off. Then it will check if your door is unlocked, and unlock it if it’s locked.
+
+## More concepts
+
+- **Zones** are geographical areas on earth. You can check if a device tracker is inside a zone as a trigger or condition in an automation. For example, “turn on the lights when Sam’s Phone comes into the ‘home’ zone”.
+
+## Integrations
+
+Home Assistant is set up to work with devices and services from lots of different companies, including voice assistants, smart devices, cameras, and other home automation software. You can make Home Assistant work with these services by installing **[integrations](https://www.home-assistant.io/integrations)**.
+
+Once installed, integrations can make new entities available, add panels to Lovelace, and more.
+
+_Formerly known as components; not to be confused with add-ons._
```

Comment:
If someone is new here, please don't mention components. It isn't used anymore, it isn't listed in the docs anymore.

Furthermore, calling out add-ons out of the blue without further text, reference or explanation is confusion. As it instantly pops the question in the reader's head: "So what is an add-on?" (which also is distracting).

Reply:
> If someone is new here, please don't mention components. It isn't used anymore, it isn't listed in the docs anymore.

In my experience as a newbie googling and such, the word has definitely come up. It might be clearer to explicitly state why so nobody things "this documentation is incomplete, what's a component...". I can remove if you feel very strongly though.

> Furthermore, calling out add-ons out of the blue without further text, reference or explanation is confusion. As it instantly pops the question in the reader's head: "So what is an add-on?" (which also is distracting).

Should I say more about add-ons, or remove it? I'm confused, add ons have come up in multiple tutorials and they're right there in the HA lovelace UI... are they "under the hood" or not?

Reply:
Right now, it is listed as a loose term, so:

>  I'm confused, add ons have come up in multiple tutorials and they're right there in the HA lovelace UI... are they "under the hood" or not?

☝️  Your own statement right there in the response shows the confusion. You highlight the possible confusion in the text, but you are not solving / removing the confusion.

I think we should either clarify what they are and fix the possible confusion, or not mention it at all.